### PR TITLE
Navigation improvement

### PR DESCRIPTION
**Pull request by saadrahim**

### CODE REVIEW

File: docs/index.md
Comment (samjwu) [inline docs/index.md:1]: Might also be worth adding `how_to/gpu_aware_mpi` to the How to Guides grid card

Also, the order of "GPU Architecture" and "GPU Isolation Techniques" is swapped when comparing the TOC and the Understand ROCm grid card

May also want to rename "ROCm File Reorganization White Paper" to "Linux Folder Structure Reorganization" in the Understand ROCm grid card for consistency

**Review by samjwu — DISMISSED** [review]

File: docs/index.md
Comment (Rmalavally) [reply docs/index.md:1]: @nunnikri @frepaul Are you ok with renaming this document? This document is specific to ROCm. Is it ok to generalize? Please advise.

Comment (frepaul) [reply docs/index.md:1]: Look good for me.

Comment (Rmalavally) [reply docs/index.md:1]: @samjwu Please proceed.

**Review by samjwu — APPROVED** [review]
looks good

**Pull request merged**